components/org.wso2.openbanking.cds.identity/src/main/java/org/wso2/openbanking/cds/identity/filter/InfoSecDataPublishingFilter.java

```diff
@@ -245,4 +255,15 @@ private String extractClientId(HttpServletRequest request) {
     public void destroy() {
     }
 
+    /**
+     * Check whether data should be published for the current request.
+     *
+     * @return boolean
+     */
+    public boolean shouldPublishCurrentRequestData(ServletRequest request) {
+
+        // Check if current request is external traffic
+        return expectedExternalTrafficHeaderValue.equals(
```

Comment:
should we do `equalsIgnoreCase`?

Reply:
Fixed in https://github.com/wso2/reference-implementation-consumerdatastandards-au/pull/24/commits/5bd8bf2502b8c24f989883b4562306f313181d37